app/helpers/container_project_helper/textual_summary.rb

```diff
@@ -16,6 +16,52 @@ def textual_group_smart_management
     items.collect { |m| send("textual_#{m}") }.flatten.compact
   end
 
+  def textual_quota
+    {
+      :labels => [_("Quota Name"), _("Resource"), _("Desired"), _("Enforced"), _("Observed")],
+      :values => collect_quota_items
+    }
+  end
+
+  def collect_quota_items
+    rows = []
+    @record.container_quota_items.each do |item|
+      rows << [
+        item.container_quota.name,
+        item.resource,
+        item.quota_desired,
+        item.quota_enforced,
+        item.quota_observed,
+      ]
+    end
+    rows
+  end
+
+  def textual_limits
+    {
+      :labels => [_("Limit Name"), _("Type"), _("Resource"), _("Max"), _("Min"), _("Default Limit"),
```

Comment:
`"Limit Name"` seems redundant to me (the table itself has the "Limits" title). It could be just `"Limit"`.